src/tutorials-democracy.md

```diff
@@ -0,0 +1,157 @@
+### Democracy Tutorial
+
+In this tutorial, we show a basic flow for two proposals in the democracy module using the CLI. On proposal will be approved and the other one will be rejected.
+
+Go to the encointer-node repo and open 2 terminals:
+
+Terminal 1:
+
+```
+git clone https://github.com/encointer/encointer-node.git
+cd encointer-node && git fetch && git checkout origin/personhood-democracy-1-delivery
+cd ..
+git clone https://github.com/encointer/pallets.git
+mv pallets/ encointer-pallets
```

Comment:
maybe worth following repo naming in your working folder and patch accordingly ;-)

Reply:
i thought i was following a convention here
https://github.com/encointer/encointer-node/blob/0a2351432bd46716c2a06e7e112ee8459ef9f7b1/Cargo.toml#L14

Reply:
yeah, the problem is I never committed patches, so this never got updated after the repo renaming